cmake/onnxruntime_unittests.cmake

```diff
@@ -1126,7 +1126,14 @@ if (NOT onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
   endif()
 endif()
 
-onnxruntime_add_shared_library_module(custom_op_library ${TEST_SRC_DIR}/testdata/custom_op_library/custom_op_library.cc)
+if (onnxruntime_USE_CUDA)
+  find_package(CUDA REQUIRED)
+  include_directories("${CUDA_INCLUDE_DIRS}")
```

Comment:
Nit: Is it better if this has target scope ?

Reply:
Do not use "find_package(CUDA REQUIRED)". It will mess up things.  Do never use it in our code. Read: https://cliutils.gitlab.io/modern-cmake/chapters/packages/CUDA.html

There are two ways of locating CUDA.  They can't co-exist.  You are using the old one.